core/src/components/datetime/datetime.tsx

```diff
@@ -1890,7 +1891,31 @@ export class Datetime implements ComponentInterface {
       <div class="calendar-header">
         <div class="calendar-action-buttons">
           <div class="calendar-month-year">
-            <ion-item button detail={false} lines="none" onClick={() => this.toggleMonthAndYearView()}>
+            <ion-item
+              ref={(el) => (this.monthYearToggleItemRef = el)}
+              button
+              aria-label="Show year picker"
+              detail={false}
+              lines="none"
+              onClick={() => {
+                this.toggleMonthAndYearView();
+                /**
+                 * TODO: FW-3547
+                 *
+                 * Currently there is not a way to set the aria-label on the inner button
+                 * on the `ion-item` and have it be reactive to changes. This is a workaround
+                 * until we either refactor `ion-item` to a button or Stencil adds a way to
+                 * have reactive props for built-in properties, such as `aria-label`.
+                 */
+                if (this.monthYearToggleItemRef) {
+                  const btn = this.monthYearToggleItemRef.shadowRoot?.querySelector('.item-native');
```

Comment:
Can we make this a const to avoid the additional access?

```js
const { monthYearToggleItemRef } = this;

if (monthYearToggleItemRef) {
  const btn = monthYearToggleItemRef.shadowRoot?.querySelector('.item-native');
...
}
```